sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job_service.py

```diff
@@ -33,14 +34,14 @@ def __init__(
         self,
         *,
         endpoint: Optional[str] = None,
-        job_service_type: Optional[str] = None,
+        job_service_type: Optional[Literal["jupyter", "jupyterlab", "ssh", "tensorboard"]] = None,
```

Comment:
have a question about `job_service_type` in general. Should it just be `type`? All other objects with `type` discriminator fields just have `type` instead of prefixing it with the entity.

Reply:
This was discussed and agreed to move for the interactive services public preview with job_service_type which already GA'ed for rest of the services in the "services" section.